WebRoot/WEB-INF/tags/calendar/monthView.tag

```diff
@@ -124,7 +124,7 @@
                                 </c:when>
                                 <c:otherwise>
                                     <app:calendarUrl var="monthZoomUrl" view="month" zoom="${true}" timezone="${timezone}" rawdate="${currentDay}"/>
-                                    <td width="11%" onclick='javascript:dayClick(event, "${monthZoomUrl}");' class='ZhCalMonthDay${currentDay.timeInMillis eq date.timeInMillis ? 'Selected':''}'>
+                                    <td width="11%" onclick='javascript:dayClick(event, "${fn:escapeXml(monthZoomUrl)}");' class='ZhCalMonthDay${currentDay.timeInMillis eq date.timeInMillis ? 'Selected':''}'>
```

Comment:
I think we need to check if fn:escapeXml is proper function or zm:cook

Reply:
I see in many places in tags file we are using fn:escapeXml, so let's go ahead with that as a safer bet

Reply:
FYI:
* zm:cook  https://github.com/Zimbra/zm-mailbox/blob/develop/common/src/java/com/zimbra/common/util/StringUtil.java#L769
* fn:escapeXml  https://github.com/javaee/jstl-api/blob/master/impl/src/main/java/org/apache/taglibs/standard/tag/common/core/Util.java#L118

Reply:
Generally, escapeHtml is considered much safer than escapeXml.
escapeXml only supports escaping the five basic XML entities (gt, lt, quot, amp, apos) whereas escapeHtml supports escaping all known [HTML 4.0 entities](http://www.w3.org/TR/REC-html40/sgml/entities.html).

In this instance, escapeXML was only chosen based on the previous code I saw.

Reply:
@jholder85638 `zm:cook` or `com.zimbra.common.util.StringUtil#escapeHtml` removes the 5 characters plus CR and LF.